configs/next/deb/monolithic/focal_dh_change_dest.patch

```diff
@@ -0,0 +1,122 @@
+# Copyright 2020 IBM Corporation
```

Comment:
Where is this patch being applied? I couldn't find it.

Reply:
It's applied by the script `scripts/utilities/pkg_build_deb.sh`